.eslintrc.js

```diff
@@ -1,15 +1,16 @@
 /** @type {import('@types/eslint').Linter.BaseConfig} */
 module.exports = {
   extends: [
-    "@remix-run/eslint-config",
-    "@remix-run/eslint-config/node",
-    "@remix-run/eslint-config/jest-testing-library",
-    "prettier",
+    'eslint:recommended',
+    '@remix-run/eslint-config',
+    '@remix-run/eslint-config/node',
+    '@remix-run/eslint-config/jest-testing-library',
+    'prettier',
   ],
   env: {
-    "cypress/globals": true,
+    'cypress/globals': true,
```

Comment:
Since we are removing `cypress` maybe it makes sense to remove it from this file?

Reply:
I would suggest to remove those too

Reply:
I would remove this because if the cypress replace PR will have to, it will cause a conflict. Easier to do here and now